.github/workflows/delivery/chocolatey/pack.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>pack</id>
+    <version>{{PACK_VERSION}}</version>
+    <packageSourceUrl>https://github.com/buildpacks/pack</packageSourceUrl>
+    <owners>Cloud Native Buildpack Authors</owners>
+
+    <title>pack</title>
+    <authors>Cloud Native Buildpack Authors</authors>
+    <projectUrl>https://github.com/buildpacks/pack</projectUrl>
+    <iconUrl>https://rawcdn.githack.com/buildpacks/artwork/36f02fae98ed82bb3175918796b6cca7acb813df/light-background/logo-light.png</iconUrl>
+    <licenseUrl>https://github.com/buildpacks/pack/blob/main/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/buildpacks/pack</projectSourceUrl>
+    <docsUrl>https://buildpacks.io/docs/</docsUrl>
+    <mailingListUrl>cncf-buildpacks@lists.cncf.io</mailingListUrl>
+    <bugTrackerUrl>https://github.com/buildpacks/pack/issues</bugTrackerUrl>
+    <tags>pack cloud-native-buildpacks cncf</tags>
+    <summary>pack is a CLI for building apps using Cloud Native Buildpacks.</summary>
+    <description>{{README}}
+    </description>
```

Comment:
I found that the `sed` wouldn't replace the `{{README}}` string normally if I had the description close on the same line; instead, it would insert on the next line, outside of the `<description>` tag entirely.

Reply:
`sed` is more foreign to me than standard RegEx so I've avoided using it. This here just adds to my avoidance. :)

Reply:
I totally agree. I just couldn't get it to work otherwise, so...